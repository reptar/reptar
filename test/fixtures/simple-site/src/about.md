---
title: About
slug: about
date: 2015-01-01T12:12:12.123Z
---

Spinning stories...

It's all in a day's work.