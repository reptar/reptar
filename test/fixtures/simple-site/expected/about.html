<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />

  <title>My Site Title</title>
  <meta name="description" content="Your website's description goes here.
">
  <meta name="keywords" content="" />

  <meta name="HandheldFriendly" content="True" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />

  <link rel="shortcut icon" href="/favicon.ico">

  <link rel="stylesheet" type="text/css" href="/css/main.css" />
</head>
<body class="post-template">

  <header class="main-header">
    <div class="vertical">
      <div class="main-header-content">
        <h1 class="page-title">
          <a href="/">My Site Title</a>
        </h1>
      </div>
    </div>
  </header>

  <main id="content" class="content" role="main">
    

<article class="post">

  <header class="post-header">
    <h1 class="post-title">About</h1>
  </header>

  <section class="post-content">
    <p>Spinning stories...</p>
<p>It's all in a day's work.</p>

  </section>

</article>



    
  </main>

  <footer class="site-footer clearfix">
    <p class="footer-description">
      &copy; My Site Title
    </p>
    <p class="footer-published">
      joyfully published by <a href="https://github.com/reptar/reptar">Reptar</a>
    </p>
  </footer>

  <script type="text/javascript" src="/js/main.js"></script>
</body>
</html>
